angular-strap/angular-strap-tests.ts

```diff
@@ -0,0 +1,378 @@
+/// <reference path="../angularjs/angular.d.ts"/>
```

Comment:
### General Comments

For anyone reviewing this pull request, you should know that I've had minimal hands-on experience with AngularStrap. There are a number of areas where I was unsure what the type definition should be. Any feedback is therefore welcome!

All tests I wrote use the examples from the AngularStrap website. Most modules only have an example for how to use the provider though, meaning that the services for most modules don't have a test at the moment. I found that there is very little documentation about services. I dug into the source code for most of them and tried my best to define all of the members that I think are meant to be public.